app/Repositories/UserRepository.php

```diff
@@ -112,6 +113,23 @@ public function counts()
     }
 
     /**
+     * Get blog authors, their number of blogs, the last created date, and title of their latest blog entry.
+     *
+     * @return Illuminate\Database\Eloquent\Builder
+     */
+    public function getAuthorPostsWithOrder()
```

Comment:
It's definitely `ASC` order, was just thinking `AuthorsPosts` doesn't sound nice haha